app/scripts/controllers/main.js

```diff
@@ -93,6 +108,27 @@ app.controller('MainCtrl', [
       $scope.autoShow = !$scope.autoShow
     }
 
+    $scope.enableNotifications = function () {
+      if (window.Notification && window.Notification.permission === 'granted') {
+        window.alert('To disable notifications, revoke the permissions in your browser.')
```

Comment:
Instead of this alert, can we instead set `$scope.webNotifications = false`? This context will of course be lost when the browser window is closed, but it might be a better workflow. Perhaps if this was coupled with some setting storage in `localStorage` that may provide the best experience.

Reply:
Interesting, hadn't thought about that. So instead of revoking permissions, you just silence the notifications. Great idea. The silence setting has to be stored preferrably. I as user found it already annoying to toggle the "autoShow" each time.

Reply:
Thanks for the input - all settings will be persisted after #291 :)